src/voronoi.rs

```diff
@@ -995,14 +939,29 @@ mod test {
 
         let mut mask = vec![false; 125];
         mask[62] = true;
-        let voronoi = Voronoi::build_partial(&generators, &mask, anchor, width, 3, true);
-        let integrator = VoronoiIntegrator::build(&generators, Some(&mask), anchor, width, 3, true);
+        let voronoi =
+            Voronoi::build_partial(&generators, &mask, anchor, width, Dimensionality::ThreeD, true);
+        let integrator = VoronoiIntegrator::build(
+            &generators,
+            Some(&mask),
+            anchor,
+            width,
+            Dimensionality::ThreeD,
+            true,
+        );
 
         let area_centroids = integrator.compute_face_integrals::<AreaCentroidIntegrator>();
         let volume_centroids = integrator.compute_cell_integrals::<VolumeCentroidIntegrator>();
 
         assert_eq!(area_centroids.len(), 1);
-        for (i, AreaCentroidIntegrator { area, centroid }) in area_centroids[0].iter().enumerate() {
+        for (
+            i,
+            AreaCentroidIntegrator {
+                area,
+                centroid,
+            },
+        ) in area_centroids[0].iter().enumerate()
+        {
```

Comment:
In my opinion this is way less legible. Again: with the default configuration, `rustfmt` does not perform this change for me...

Reply:
Yeah, I agree.
I just saw I had `use_small_heuristics = "Off"` in there, for no good reason.

I used this (I copied it from another crate since I wanted the doc/imports stuff and somehow forgot to remove the bit at the top):
```toml
# See https://rust-lang.github.io/rustfmt/?#use_small_heuristics
use_small_heuristics = "Off"
# Comments -----------------------------------------------------------
wrap_comments = true
format_code_in_doc_comments = true
# Imports ------------------------------------------------------------
imports_granularity = "Crate"
```